### DISCUSSION TITLE
how to protect certain data

### DISCUSSION THREAD

**Discussion by FifthWit** [pocketbase/pocketbase#5672] [Q&A]
I want to have a feature to see other users for my app, but I dont want other users to be able to see each others emails without being the account owner, so other users can see each others username, avatar but not thigns like the email, how can I protect that with api rules?

**Comment by tigawanna** [comment]
email visibility can be toggled in auth collections see : https://github.com/pocketbase/pocketbase/discussions/3668#discussioncomment-7467354
